stable/openldap/ccp_values.yaml

```diff
@@ -0,0 +1,127 @@
+# Default values for openldap.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# Define deployment strategy - IMPORTANT: use rollingUpdate: null when use Recreate strategy.
+# It prevents from merging with existing map keys which are forbidden.
+strategy: {}
+  # type: RollingUpdate
+  # rollingUpdate:
+  #   maxSurge: 1
+  #   maxUnavailable: 0
+  #
+  # or
+  #
+  # type: Recreate
+  # rollingUpdate: null
+image:
+  # From repository https://github.com/osixia/docker-openldap
+  repository: osixia/openldap
```

Comment:
do you trust Osixia user? Don't you think would be better to use openshift's one? 
@mlimaloureiro not sure what is the process here, but I remember someone saying that he was "digging" inside an image to validate what was running inside.

Reply:
@jsaguiar we need to use specifically this image because helm chart is based on it and uses its specific configuration. We discussed image and chart selection [here](https://confluence.kigroup.de/pages/viewpage.action?pageId=15763094).
This image is definitely not a good choice, but it's very configurable and flexible. And I can say, that I already "dug" into it and have an understanding of what's inside. Also, from openshift image README:
`DISCLAIMER - IMAGES ONLY FOR INTERNAL TESTING OF ORIGIN REPOSITORY`

Looking on it now, and it has so poor options to customize which is needed(or may be needed) for our setup. For example for importing custom schemas, defining ACL e.t.c.